### DISCUSSION TITLE
UK government transparency framework

### DISCUSSION THREAD

**Discussion by Joe-Heffer-Shef** [pykale/transparentML#27] [General Sharing]
This is the framework I raised in the class:

["Ethics, Transparency and Accountability Framework for Automated Decision-Making"](https://www.gov.uk/government/publications/ethics-transparency-and-accountability-framework-for-automated-decision-making/ethics-transparency-and-accountability-framework-for-automated-decision-making)

> The ethical considerations of artificial intelligence and automated systems is at the centre of technological advancement.
>
> According to a recent [EU survey](http://www.beuc.eu/publications/beuc-x-2020-078_artificial_intelligence_what_consumers_say_report.pdf?utm_source=POLITICO.EU&utm_campaign=5a7d137f82-EMAIL_CAMPAIGN_2020_09_09_08_59&utm_medium=email&utm_term=0_10959edeb5-5a7d137f82-190598416) and a [British Computer Society survey](https://www.bcs.org/more/about-us/press-office/press-releases/the-public-dont-trust-computer-algorithms-to-make-decisions-about-them-survey-finds/) in the UK, there is a distinct distrust in the regulation of advanced technology. A [review by the Committee on Standards in Public Life](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/868284/Web_Version_AI_and_Public_Standards.PDF) found that the government should produce clearer **guidance on using artificial intelligence ethically in the public sector**.

**Comment by Joe-Heffer-Shef** [comment]
IBM research [A framework for fostering transparency in shared artificial intelligence models by increasing visibility of contributions](https://research.ibm.com/publications/a-framework-for-fostering-transparency-in-shared-artificial-intelligence-models-by-increasing-visibility-of-contributions)

**Comment by Joe-Heffer-Shef** [comment]
[6 Explainable AI (XAI) Frameworks for Transparency in AI](https://dev.to/amananandrai/6-explainable-ai-xai-frameworks-for-transparency-in-ai-3koj)

**Comment by richplant** [comment]
Rudin (2018) argues that there's a moral responsibility to stop using explanatory investigation for black boxes, and instead use interpretable models: [Stop Explaining Black Box Machine Learning Models for High Stakes Decisions and Use Interpretable Models Instead](https://arxiv.org/abs/1811.10154)
